### PR TITLE
Additional wrappers to SLICOT functions

### PR DESCRIPTION
**Pull request by mdclemen**
I've added wrappers to SLICOT functions SB03OD and AB09BD. I have an open pull request on jgoppert/Slycot, but that fork seems to be abandoned.

### CODE REVIEW

**Comment by slivingston** [comment]
For posterity, the prior pull request that is referenced in the opening post is https://github.com/jgoppert/Slycot/pull/8

File: slycot/src/synthesis.pyf [hunk 1]
```diff
@@ -410,13 +430,13 @@ subroutine sb10ad(job,n,m,np,ncon,nmeas,gamma,a,lda,b,ldb,c,ldc,d,ldd,ak,ldak,bk
     double precision intent(out), dimension(ncon,nmeas), depend(ncon,nmeas) :: dk
     integer intent(hide), depend(dk) :: lddk=shape(dk,0)
     double precision intent(out), dimension(2*n,2*n), depend(n) :: ac
-    integer intent(hide), depend(ac) :: ldac=shape(ac,0)
+    integer intent(hide), depend(ac) :: ldak=shape(ac,0)
```
Comment (roryyorke) [inline slycot/src/synthesis.pyf]: this (and the other changes to sb10ad) undo the fix in 42cdeb82 -- this is why the unit test fails.

**Comment by slivingston** [comment]
@mdclemen Can you rebase to address the following?
* remove the text "First commit of mdc-dev branch." from the first commit, which is not meaningful in the commit message because the branch ref will likely eventually be lost, and the particular branch name is not relevant to understanding the changeset;
* delete the part of your patch that undoes work by @roryyorke in 42cdeb829cc42c8e21faf1af09110533f1e1ac18, as commented in https://github.com/python-control/Slycot/pull/1#discussion_r75529621
* remove the commit that has message that begins as "Merge branch 'master' of https://github.com/mdclemen/Slycot-1..."

File: slycot/analysis.py [hunk 2]
```diff
@@ -876,7 +876,7 @@ def ab09bd(dico,job,equil,n,m,p,A,B,C,D,nr=None,tol1=0,tol2=0,ldwork=None):
         e = ArithmeticError('The computation of Hankel singular values failed')
         e.info = out[-1]
         raise e
-    nr,A,B,C,D,hsv = out[:-2]
+    Nr,A,B,C,D,hsv = out[:-2]
     return nr, A[:Nr,:Nr], B[:Nr,:], C[:,:Nr],D[:,:], hsv
```
Comment (roryyorke) [inline slycot/analysis.py]: should that be `return Nr,` [etc.] ?  (This is just by comparison with ab09ad, which returns Nr, not nr.)

**Comment by mdclemen** [comment]
@slivingston, I think I have done the things you have asked. My git status is novice at best.

**Comment by slivingston** [comment]
I can assist with rebasing or modifying changesets via email or on IRC provided by [OFTC](https://www.oftc.net/). For the latter, there is a web client that you can use: <https://webchat.oftc.net/?channels=#python-control>.

Also, we can simply avoid any modifications to commits during review and then decide what to do when the final, overall patch is good to go.

**Comment by mdclemen** [comment]
I closed this pull-request and started a new one with a clean commit.

**Pull request closed without merging**